### ISSUE TITLE
parsing time "" as "2006-01-02T15:04:05Z07:00": cannot parse "" as "2006"

### ISSUE THREAD

**Issue by BloodStainedCrow** [Mic92/sops-nix#585]
I am following your Usage Example Section and when running `nix-shell -p sops --run "sops secrets/firefly/.db.env"`,
I get the cryptic error: `parsing time "" as "2006-01-02T15:04:05Z07:00": cannot parse "" as "2006"`.

This error might originate from sops itself.
Any help is hugely appreciated!

**Comment by Mic92** [comment]
Looks like it doesn't like the lastmodified field of your encrypted file. This is what it looks like: https://github.com/Mic92/dotfiles/blob/9cbaa68b64bb584b30791d9297d5eef89fa2a99f/sops/secrets/alertmanager/secret#L22

Is .db.env actually an encrypted file?

**Comment by BloodStainedCrow** [comment]
No. I tried to encrypt an already existing file, which is apparently is wrong. Instead I should run `nix-shell -p sops --run "sops secrets/firefly/.db.env"` when the file does not yet exist.

Could it be possible to improve the generated error message? Or is this a sops problem not a sops-nix one?
In that case I would open an issue there
